content/guides/vercel-sdk-migration.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Migrate from Vercel Postgres SDK to the Neon serverless driver
+subtitle: Learn how to smoothly transition your Next.js application from the Vercel Postgres SDK to the Neon serverless driver
+author: dhanush-reddy
+enableTableOfContents: true
+createdAt: '2024-10-28T00:00:00.000Z'
+updatedAt: '2024-10-28T00:00:00.000Z'
+---
+
+With Vercel Postgres transitioning to Neon's native integration in the [Vercel Marketplace](https://vercel.com/blog/introducing-the-vercel-marketplace), now is the perfect time to migrate from the Vercel Postgres SDK (@vercel/postgres) to the Neon serverless driver. This guide will walk you through the migration process, helping you leverage the full power of Neon's serverless PostgreSQL capabilities.
+
+## Why migrate?
+
+Switching to the Neon serverless driver offers several advantages over the Vercel Postgres SDK:
+
+- **Direct database access**: Connect directly to your database without added abstraction layers.
+- **Enhanced performance**: Benefit from Neon's optimized database driver.
+- **Greater flexibility**: Choose HTTP for single queries or WebSockets for transactions.
+- **Improved maintainability**: Rely on Neon's actively maintained, native database driver.
```

Comment:
I think we can keep the last 2 points only?

Reply:
updated